packages/augur-ui/src/modules/account/components/account-view.styles.less

```diff
@@ -140,53 +140,85 @@
 
 :root[theme="BETTING"],
 :root[theme="SPORTS"] {
-  .AccountView {
-    > div {
-      border: @size-1 solid var(--color-outline);
-      border-radius: @border-radius-rounded;
+  .AccountViewWrap {
```

Comment:
we shouldn't need `AccountViewWrap`, the HTML should change very little in `account-view.tsx` in terms of its structure. we would want ot just modify `AccountView`.

Reply:
Removed.